AK/Statistics.h

```diff
@@ -51,16 +71,26 @@ class Statistics {
         return maximum;
     }
 
-    // FIXME: Implement a better algorithm
     T const median()
```

Comment:
same issue as with the `quickselect`; both `quickselect_inplace` and `quick_sort` modify the underlying structure. i believe that we should indicate that more clearly, esp. with apparently read-only operation like `median` computation.

Reply:
We can modify `m_values` freely since it's not directly visible to the users and it doesn't change the results of any other we have implemented in statistics right now.

Since the old Median implementation modified it aswell I think it's fine to modify it in place.